chromaui/addon-visual-tests
# Only use criteria to select mode when story first loads

QA

1. Use a project with stories with modes:
   - One story should have a mode that is passed and then a mode that has changed, in that order
   - Another story should have the same modes, but both modes have changes
   - A third story should have additional modes that are not found in the first two stories, with changes
2. Navigate to the first story and verify that the second mode (the one with changes) is selected
3. Verify that you can change to the first mode and view the test
4. Navigate to the second story and verify that the same mode is shown from step 3
5. Navigate back to the first story and verify the the _unchanged_ mode is selected (this takes precedence over a selected mode that does not contain changes)
6. Navigate to the third story and select a mode that is unique to that story
7. Navigate back to the first story and verify that the mode with changes is selected

<details>
  <summary>📦 Published PR as canary version: <code>1.2.22--canary.218.31120f4.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromatic-com/storybook@1.2.22--canary.218.31120f4.0
  # or 
  yarn add @chromatic-com/storybook@1.2.22--canary.218.31120f4.0
  ```
</details>
